.travis.yml

```diff
@@ -25,15 +27,25 @@ cache:
     - $VERILATOR_CACHE
 
 before_install:
-# Perl modules needed for testing
-# Not listing Bit::Vector as slow to install, and only skips one test
-  - touch temp.cpp ; g++ -E -dM -c temp.cpp | sort ; rm -rf temp.cpp
-  - yes yes | sudo cpan -fi Unix::Processors Parallel::Forker
-  - sudo apt-get install gdb gtkwave lcov
-  - sudo apt-get install libfl-dev || true
-  - sudo apt-get install libgoogle-perftools-dev
-# Only works on 20.04
-  - sudo apt-get install libsystemc-dev || true
+  - |
+    if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      touch temp.cpp ; g++ -E -dM -c temp.cpp | sort ; rm -rf temp.cpp;
+      sudo apt-get install gdb gtkwave lcov;
+      sudo apt-get install libfl-dev || true;
+      sudo apt-get install libgoogle-perftools-dev;
+      # Only works on 20.04
+      sudo apt-get install libsystemc-dev || true;
```

Comment:
We should set VERILATOR_NUM_JOBS here instead of globals above, `nproc` above is causing an error inthe FreeBSD build